android/app/src/main/java/org/fossasia/openevent/utils/SharedPreferencesUtil.java

```diff
@@ -10,7 +10,7 @@
     private static SharedPreferences.Editor editor;
 
     static {
-        sharedPreferences = OpenEventApp.getAppContext().getSharedPreferences(ConstantStrings.FOSS_PREFS, Context.MODE_PRIVATE);
+        sharedPreferences = PreferenceManager.getDefaultSharedPreferences(OpenEventApp.getAppContext());
```

Comment:
All the settings preferences are stored in the default preferences. If we use different preference than we will not able to get values from default. Problem I have faced was at https://github.com/fossasia/open-event-android/blob/development/android/app/src/main/java/org/fossasia/openevent/OpenEventApp.java#L111 It always returns `false` if we use other preference than default preference